### PR TITLE
[WIP] Added support for erfcinv.

### PR DESCRIPTION
**Pull request by finbarrtimbers**
As requested in #8089. My machine is having errors running the tests, so this isn't tested; I would be very grateful if one of the maintainers can run the tests in Travis and I'll fix the results.

### CODE REVIEW

File: tensorflow/core/kernels/cwise_op_gpu_erfcinv.cu.cc [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```
Comment (dandelionmane) [inline tensorflow/core/kernels/cwise_op_gpu_erfcinv.cu.cc]: s/2015/2017/

File: tensorflow/core/kernels/cwise_op_erfcinv.cc [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* Copyright 2015 The TensorFlow Authors. All Rights Reserved.
```
Comment (dandelionmane) [inline tensorflow/core/kernels/cwise_op_erfcinv.cc]: s/2015/2017/

**Comment by dandelionmane** [comment]
@tensorflow-jenkins test this please

**Comment by finbarrtimbers** [comment]
Still working on this- sorry. Should have more commits soon!

**Comment by finbarrtimbers** [comment]
Has the `erfinv` op been implemented in Tensorflow anywhere? I can't seem to find it, so I'm planning on implementing my own version (following this Stack Overflow [answer](https://stackoverflow.com/questions/42381244/pure-python-inverse-error-function)).

**Comment by martinwicke** [comment]
I added [WIP] to the description. Remove that once it's ready to be looked at. Thanks!

**Comment by drpngx** [comment]
Closing to keep the set of open PRs manageable. Feel free to reopen once ready.

**Pull request closed without merging**